src/interfaces/packagingInterfacesAndType.ts

```diff
@@ -70,3 +70,39 @@ export type Package2VersionCreateEventData = {
   message?: string;
   timeRemaining?: Duration;
 };
+
+export type PackageVersionListResult = {
```

Comment:
These need exporting too right?

Reply:
@shetzel not following, the index.ts is reexporting everything in this file and so-on up to src/exported.ts.